docs/datatype/image/text2image.md

```diff
@@ -1,3 +1,484 @@
 # Search Image from Text via CLIP model
 
-TBA
\ No newline at end of file
+In this tutorial, we will create an image search system that retrieves images based on short text descriptions of their content.
+
+The interest behind this is that regular search or queries rely on image description or meta data to describe the content of the image, this is simply not efficient 
+and expensive because you need a person to write that description and also information about image content is not always available.
+
+We need to look for another solution ! What if we directly compare text with images ?  
+
+To do so, we need to figure out a way to match images and texts. One way to find such matches we need to find related images with similar semantics to the query texts. This requires us to represent both images and query texts in the same embedding space to be able to do the matching. In this case, pre-trained cross-modal models can help us out.
+
+For example when we write the word "dog" in query we want to be able to retrieve pictures with a dog in then solely by using the embeddings similarity.
+
+```{tip}
+The full source code of this tutorial is available in this [google colab notebook](https://colab.research.google.com/drive/1CkT1udBrMfefYo0XJO45kIpW43odLLDm?usp=sharing)
+```
+
+Now that we understand the problem and we have an idea on how to fix it, let's try to imagine what the solution would look like : 
+
+1. We have a bunch of images with no text description about the content or anything.
+2. We use a model to create an embedding that represents those images. 
+3. Now we will index and save our embeddings which we will call Documents inside a workspace folder. 
+
+This is what we call the index flow.  
```

Comment:
**Automated message**

## Capitalization

Please capitalize first letters of the following (when used to reference project or component):

- Flow
- Executor
- Document
- Jina
- Jina Hub
- Python
- Docker
- Kubernetes

### Note!

This only applies **outside** a code or CLI context. If you're describing Python, YAML, any programming language, or command line input/output, please use the capitalization you would use **in that context**.

##### Right

```python
from jina.flow import Flow
```

##### Wrong

```python
from Jina.Flow import Flow
```

---

Made with [GitHub saved replies](https://github.com/settings/replies)